account/guides/setupRegistrar.md

```diff
@@ -0,0 +1,147 @@
+
+Once MD5 is installed, run the command: `md5 -s {composite-username}:{Realm}:{Password}` where `{Password}` is the desired password
+
+#### Default Setup (_No Domain Specified as Part of the User's ID_)
+##### Generate MD5 `Hash1` _without_ Domain
+| composite-username | : | realm                       | : | password   |
+|--------------------|---|-----------------------------|---|------------|
+| `sipauthtest`      | : | `<randomAccountHex>.auth.bandwidth.com` | : | `password` |
+```
+md5 -s sipauthtest:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("sipauthtest:<randomAccountHex>.auth.bandwidth.com:password") = fe438bddfc087dda89d29e637f5684ab
+```
+
+##### Generate MD5 `Hash2` _without_ Domain
+| composite-username | : | realm                       | realm                       | : | password   |
+|--------------------|---|-----------------------------|-----------------------------|---|------------|
+| `sipauthtest`      | : | `<randomAccountHex>.auth.bandwidth.com` | `<randomAccountHex>.auth.bandwidth.com` | : | `password` |
+```bash
+md5 -s sipauthtest:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("sipauthtest:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password") = 79bb0e55551e14a2f329a282c7cf145
+```
+
+#### Non-Default Setup (_Domain Specified_)
+##### Generate md5 Hash1 _with_ domain `somewhere.com`
+| composite-username  | : | realm                       | : | password   |
+|---------------------|---|-----------------------------|---|------------|
+| `bob@somewhere.com` | : | `<randomAccountHex>.auth.bandwidth.com` | : | `password` |
+```bash
+md5 -s bob@somewhere.com:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("bob@somewhere.com:<randomAccountHex>.auth.bandwidth.com:password") = 817d76e91aad032a8c272229f468bfb2
+```
+
+#####  Generate md5 Hash2 _with_ domain `somewhere.com`
+| composite-username  | : | realm                       | realm                       | : | password   |
+|---------------------|---|-----------------------------|-----------------------------|---|------------|
+| `bob@somewhere.com` | : | `<randomAccountHex>.auth.bandwidth.com` | `<randomAccountHex>.auth.bandwidth.com` | : | `password` |
+```bash
+md5 -s bob@somewhere.com:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("bob@somewhere.com:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password") = 39679d2a73c2e1ea719621bc0d8fdac8
+```
+
+### Add the Newly Created Hash to Sip Credentials {#add-new-credentials}
+After generating the hash value, fill in the Hash1 and Hash2 values to create the SIP credentials.
+
+#### Using the UI  {#using-the-ui}
+Select `Sip Credentials` from the account overview page on the [Bandwidth Dashboard](dashboard.bandwidth.com)
+
+<img src="../../images/sip_credentials.png" style="max-width:95%">
+
+<img src="../../images/create_sip_credentials.png" style="max-width:95%">
```

Comment:
This image needs to be updated - we no longer allow Domain changes from the UI, so mentions of domain from the API perspective are good to leave in but this screenshot should be changed.

Reply:
It also should show the `Realm` and `Application` textboxes.

Reply:
Fixed.